builtin/fstk/ui.lua

```diff
@@ -78,37 +78,48 @@ end
 
 --------------------------------------------------------------------------------
 function ui.update()
-	local formspec = ""
+	local formspec = {}
 
 	-- handle errors
 	if gamedata ~= nil and gamedata.reconnect_requested then
-		formspec = wordwrap_quickhack(gamedata.errormessage or "")
-		formspec = "size[12,5]" ..
-				"label[0.5,0;" .. fgettext("The server has requested a reconnect:") ..
-				"]textlist[0.2,0.8;11.5,3.5;;" .. formspec ..
-				"]button[6,4.6;3,0.5;btn_reconnect_no;" .. fgettext("Main menu") .. "]" ..
-				"button[3,4.6;3,0.5;btn_reconnect_yes;" .. fgettext("Reconnect") .. "]"
+		local error_message = core.formspec_escape(
+				gamedata.errormessage or "<none available>")
```

Comment:
Should `"<none available>"` be translatable? Message below also.

Reply:
For a placeholder this does not really matter. The error messages are likely to be in English anyway.